playbooks/roles/forum/defaults/main.yml

```diff
@@ -23,8 +23,9 @@ FORUM_SINATRA_ENV: "development"
 FORUM_RACK_ENV: "development"
 FORUM_NGINX_PORT: "18080"
 FORUM_API_KEY: "password"
-FORUM_ELASTICSEARCH_HOST: "localhost"
-FORUM_ELASTICSEARCH_PORT: "9200"
+
+FORUM_ELASTICSEARCH_HOST: "search-edx-stage-3lznpjrtxtyqednz3brtuycxmy.us-east-1.es.amazonaws.com"
```

Comment:
We'll need a way to differentiate the staging vs production instance. Seeing as these are specified "defaults", surely there is a way to override per environment. I will investigate.